src/OrchardCore/OrchardCore.Data.YesSql/DatabaseHelper.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace OrchardCore.Data;
+internal class DatabaseHelper
+{
+    public static string GetStandardPrefix(string prefix)
```

Comment:
Is it necessary? Maybe it can be retrieved from the `YesSql.Configuration`?
Just some thought.

Reply:
@Skrypt I think so. the same logic is used into multiple places and it isn't int `YesSql.Configuration` it is being defined in `[OrchardCoreBuilderExtensions.cs](https://github.com/OrchardCMS/OrchardCore/pull/11822/files/aa461200d3106c54f93ede74cd7c5e7c06a741c9#diff-b499711d6aba6467e68b5d1377792bdadf5074011067c7ecedfd8e166458ec70) I thought it would be best to abstract that into common helper so it defined in one place and used multiple places

Reply:
Ok, but if someone uses a different prefix with the OrchardCoreBuilderExtension then this won't work for him.

Reply:
@Skrypt yes I get it. But the `YesSql.Configuration` does not store the separator `_` separately. Int he validator we need to know the used separator to be able to see if the table `prefix` is already in use or not.

Another way we could add a new property called `PrefixSeparator` to new YesSqlOptions  with `_` being the default. Then we could remove the Helper method and instead `prefix + YesSqlOptions.PrefixSeparator ?? String.Empty`. Do you want me to make this change?

Reply:
@Skrypt check out the change now. I removed the DatabaseHelper. Now `TablePrefixSeparator` is an option that is available on `YesSqlOptions`. During the site setup, we store `TablePrefixSeparator` in the site settings so if the `YesSqlOptions.TablePrefixSeparator` change after tenants were setup, it wont break anything. For backward compatibility, if the TablePrefixSeparator isn't set, we use _ as the default value.

Let me know if you see an issue or requesting more changes.

Reply:
Ok, I like it.